src/site/content/en/blog/test-post/index.md

```diff
@@ -602,6 +602,16 @@ quis enim blandit, posuere justo dignissim, scelerisque diam. Fusce aliquet urna
 ac blandit ullamcorper. Proin et semper nibh, sit amet imperdiet velit. Morbi at
 quam sem.
 
+{% Widget 'example-set/graph', 520 %}
```

Comment:
It's not a big deal, but `520` is too small for the sample content—the "Highcharts.com" text gets cut off. `540` seems fine.

<img width="346" alt="Screen Shot 2022-03-09 at 11 09 54 AM" src="https://user-images.githubusercontent.com/1749548/157482980-1977b605-351c-4f0f-ab6a-88c7f86f2316.png">